testsuites/syncgateway/functional/tests/test_db_online_offline.py

```diff
@@ -567,8 +598,16 @@ def test_offline_true_config_bring_online(params_from_base_test_setup, sg_conf_n
 
     # Scenario 9
     # POST /db/_online
-    status = admin.bring_db_online(db="db")
-    assert status == 200
+    # Take bucket offline
+    ansible_runner = AnsibleRunner(cluster_conf)
+    status = ansible_runner.run_ansible_playbook(
```

Comment:
I feel like we should wrap this up in an API call. Ex. `sg_client.db_offline(sg_admin_url, db)`